### PR TITLE
Update omv-btrfs-scrub: only send mail if the scrub command found an error 

### PR DESCRIPTION
**Pull request by mael2**
Update omv-btrfs-scrub: only send mail if the scrub command found an error
try to reduce mails from OMV.
PIPESTATUS only works in bash.
Signed-off-by: Florian Schröck [mael2@mailbox.org](mailto:mael2@mailbox.org)

This is my first pull request ever.
Let me know, if there is something to improve :)

### CODE REVIEW

**Comment by ryecoaaron** [comment]
In my opinion, this should be configurable not error-only.  Some users will want to receive an email on success.

**Comment by mael2** [comment]
I didn't dive into the project this deep yet, but that would be great!

**Comment by votdev** [comment]
> In my opinion, this should be configurable not error-only. Some users will want to receive an email on success.

I'm also in favour of sending emails when the scan is suceeded, not only if it fails.

**Comment by votdev** [comment]
Please upgrade the PR and subscribe the DCO.

File: deb/openmediavault/usr/sbin/omv-btrfs-scrub [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env dash
+#!/bin/bash
```
Comment (votdev) [inline deb/openmediavault/usr/sbin/omv-btrfs-scrub:1]: Please use the path returned by `which`.
```suggestion
#!/usr/bin/bash
```

**Review by votdev — CHANGES_REQUESTED** [review]

**Comment by mael2** [comment]
I don't understand how to add the signoff to a single commit where it's missing. I think I made it worse. let's forget about it.

**Comment by ryecoaaron** [comment]
@mael2 just use the --signoff flag when you commit a change.  If you want to sign an existing commit, just use git commit --amend --signoff

**Pull request closed without merging**